webroot/adm/enclave-gcp-v2.html

```diff
@@ -0,0 +1,173 @@
+<html>
+
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="/js/main.js"></script>
+  <style>
+    label.bold {
+      font-weight: bold
+    }
+  </style>
+</head>
+
+<body>
+<h1>UID2 Admin - GCP Enclave ID Tool(V2)</h1>
+
+<a href="/">Back</a>
+
+<br>
+<br>
+
+<h3>Inputs</h3>
+
+<label for="imageId" class="bold">Operator Docker Image ID:</label>
+<br>
+<label for="imageId">The full digest for the image, with or without 'sha256:'</label>
+<br>
+<input type="text" id="imageId" name="imageId" size="100" value="">
+<br>
+<br>
+
+<label for="envId" class="bold">Environment:</label>
+<br>
+<select id="envId">
+  <option value="integ">Integration</option>
+  <option value="prod">Production</option>
+</select>
+<br>
+<br>
+
+<label for="identityScope" class="bold">Identity Scope:</label>
+<br>
+<select id="identityScope">
+  <option value="uid2">UID2</option>
+  <option value="euid">EUID</option>
+</select>
+<br>
+<br>
+
+<label for="debugMode" class="bold">DebugMode:</label>
+<br>
+<label for="debugMode">
+  Set this to true if you want to use DebugMode Confidential Space host image in integration env.
+  (You need to generate a new enclaveId.)
+</label>
+<br>
+<select id="debugMode">
+  <option value="false">False</option>
+  <option value="true">True</option>
+</select>
+<br>
+<br>
+
+<h3>Operations</h3>
+
+<ul>
+  <li><a href="#" id="doGenerate">Generate Enclave ID and gcloud command</a></li>
+</ul>
+<br>
+
+<h3>Outputs</h3>
+
+<label for="gcloudCli">Example gcloud command to bring up Confidential Space VM:</label>
+<br>
+<textarea id="gcloudCli" name="gcloudCli" rows="10" cols="120" readonly>
+</textarea>
+<br>
+<br>
+
+<label for="enclaveId">Generated GCP Enclave ID:</label>
+<br>
+<pre><label id="enclaveId">TBD</label></pre>
+<br>
+<br>
+
+<script language="JavaScript">
+  const gcloudCli = '$ gcloud compute instances create <INSTANCE_NAME> \\\n\
```

Comment:
Should we add a placeholder for zone config so that partners can change it?

Reply:
Also for `machine-type`

Reply:
There would be too many GCP overrides. I would suggest to only provide necessary commands for partners. i.e. they will use default zone & machine-type if not specified, but OK to change. We could mention this in public doc guide.
WDYT?

Reply:
I agree, we can document what else they can change

Reply:
I see, a well written doc will be good enough